packages/Babylonian-Tests.package/BPExampleToMethodTestTest.class/instance/testTest.st

```diff
@@ -0,0 +1,5 @@
+tests
+testTest
+	
+	self assert: ('testExample<n><t>"Generated from an example, do not modify"<n><n><t>self assert: ((Thing new) doBla: 2 with: 42) equals: (3)' expandMacros)
```

Comment:
hardcoded value "generated at..." what already is given by `BPExampleToTest >testComment`